tests/nested/core/core20-kernel-reseal/task.yaml

```diff
@@ -16,9 +19,8 @@ prepare: |
   # ensure we really have the header we expect
   grep -q -a "This program cannot be run in DOS mode" pc-kernel/kernel.efi
   # modify the kernel so that the hash changes
-  sed -i 's/This program cannot be run in DOS mode/This program cannot be run in XXX mode/' pc-kernel/kernel.efi
-  grep -q -a "This program cannot be run in XXX mode" pc-kernel/kernel.efi
-
+  llvm-objcopy --add-section .modified=/dev/null pc-kernel/kernel.efi
```

Comment:
Why is the previous method for this not working? llvm is quite a big package, so it would be better to avoid installing it.

Reply:
I have updated the test, this is just needed for uc24 tests because the following line fails
grep -q -a "This program cannot be run in XXX mode" pc-kernel/kernel.efi

Reply:
I guess that is because now kernel.efi is built with a systemd tool which does not add the usual PE headers. So it si fine, although I guess we could do some other modifications to the binary.